docs/extend/extend-apiml/onboard-wizard.md

```diff
@@ -1,12 +1,18 @@
 # Onboarding a REST API service with the YAML Wizard
-## Description
-The purpose of this wizard is to simplify the process of onboarding new REST API services to the Zowe API Mediation Layer. It offers a walkthrough of the required steps to create a correct configuration file which is used to set the application properties and eureka metadata.
+
+As an API developer, you can use the Yaml Onboarding Wizard to simplify the process of onboarding new REST API services to the Zowe API Mediation Layer. The wizard offers a walkthrough of the required steps to create a correct configuration file which is used to set the application properties and Eureka metadata.
+
 ## Onboarding your REST service with the Wizard
-1. Click on the 'Onboard New API' dropdown located in the navbar  
+
+The following procedure describes how to onboard your REST service with the Wizard.
+
+**Follow these steps:**
+
+1. In the ... , click the 'Onboard New API' dropdown located in the navbar.  
```

Comment:
Where is the user in this first step?

Reply:
Main page, the button is in the navbar. We will add that to the doc.

Reply:
Main page?

Reply:
Main page of the API catalog. The first thing the user sees when he logs in - the dashboard.